lang/csharp/src/apache/test/CodGen/CodeGenTest.cs

```diff
@@ -30,7 +30,7 @@ namespace Avro.Test
 
     class CodeGenTest
     {
-#if !NETCOREAPP2_2 // System.CodeDom compilation not supported in .NET Core: https://github.com/dotnet/corefx/issues/12180
+#if !NETCOREAPP3_1 // System.CodeDom compilation not supported in .NET Core: https://github.com/dotnet/corefx/issues/12180
```

Comment:
These can be changed to just `#if !NETCOREAPP`. That way they don't have to be updated every time the TFM changes.

Reply:
That is correct. I wanted to keep the change to be as similar to the original as possible, however the intent is to differentiate between netcoreapp and not netcoreapp.